src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -586,13 +586,16 @@ export class AppComponent implements OnInit, AfterViewChecked {
     /**
      * Format all text to replace all new lines with spaces and performs HTML entity encoding
      */
-    textFormatter(row: number, cell: any, value: string, columnDef: any, dataContext: any): string {
-        let valueToDisplay = value;
+    textFormatter(row: number, cell: any, value: any, columnDef: any, dataContext: any): string {
```

Comment:
The hyperlink formatter should be corrected as per your request. Interestingly, I didn't find any unit tests that check the function of the hyperlink formatter